tools/tests.py

```diff
@@ -271,7 +266,7 @@
         "id": "MBED_A20", "description": "I2C master/slave test",
         "source_dir": join(TEST_DIR, "mbed", "i2c_master_slave"),
         "dependencies": [MBED_LIBRARIES, TEST_MBED_LIB,],
-        "mcu": ["LPC1768", "RZ_A1H", "NUCLEO_F411RE", "NUCLEO_F446RE", "NUCLEO_F429ZI", "DISCO_F429ZI", "NUCLEO_F446ZE",  "NUCLEO_F410RB", "NUCLEO_F401RE"],
+        "automated": True,
```

Comment:
`automated` just means that the test will be ran automatically if no test names are specified.